docs/validators/approval.rst

```diff
@@ -18,6 +18,7 @@ Approvals
         message: 'Custom message...'
       limit:
         teams: ['org/team_slug'] # when the option is present, only the approvals from the team members will count
+        users: ['user1', 'user2'] # when the option is present, approvals from users in this list will count
```

Comment:
I'm not sure this comment is clear:

1. Does it mean that only these users listed will be summed as the total required approval count?
1. Does it mean that only this subset of users from the team members will count?

I believe the documentation needs work and a comment in this example config is not enough.

I suggest a more detailed description outside of a settings sample.

Reply:
It's option 1 which you mentioned

+1, the documentation can be improved

Reply:
added a note on the page to try and clarify it.